views/js/e2e/_helpers/commands/setupCommands.js

```diff
@@ -83,17 +178,47 @@ Cypress.Commands.add('publishTest', (testName) => {
         cy.contains(testName).click({ force: true });
     });
 
-    // Publish example test
+    cy.wait('@editTest');
+
+    // Publish selected test
     cy.get(setupSelectors.testsPage.testPublishButton).click();
 
+    if (deliveryType === 'remote') {
+        // Selects TAO Remote tab
+        cy.get(setupSelectors.testsPage.deliveryTypeTabs).contains('TAO Remote').click();
+    }
+
+    // Select suitable tenant
+    cy.get('.deliver-tenant-list').select('3');
+
     // Select Assembled Delivery as root class for publishing
     cy.get(setupSelectors.testsPage.destinationSelector).contains('Assembled Delivery').click();
 
     // Clicking on publish
     cy.get(setupSelectors.testsPage.destinationSelectorActions).contains('Publish').click();
 
-    //Wait until test is published
-    cy.wait('@testPublish');
+    // Wait until test is published
+    if (deliveryType === 'remote') {
+        cy.wait('@publishDeliverConnect');
+    }
+    else {
+        cy.wait('@publishDeliveryRdf');
+    }
+    /*
+     * Two different processes to handle different UX
+     * for taoTaskQueue enabled or disabled
+     */
+    cy.get('@isTaskQueueWorking').then(working => {
+        if (working) {
+            cy.wait('@taskQueueWebApi'); // can be 1-4 of them :(
+
+            pollTaskQueue({ taskLabel: 'Remote Publication Lookup', status: 'completed' });
```

Comment:
`Remote Publication Lookup` won't happen when you do a local publication.

Reply:
Now wrapped in `if`.